.github/workflows/rubocop.yml

```diff
@@ -49,10 +52,20 @@ jobs:
           git clone https://github.com/rubocop-hq/rubocop.git ../rubocop
           chmod +x ../rubocop/exe/rubocop
           cd ../rubocop && bundle install --jobs 3 --retry 3
+      - name: code coverage
+        if: matrix.coverage == 'yes'
+        uses: paambaati/codeclimate-action@v2.6.0
+        env:
+          CC_TEST_REPORTER_ID: '758a8228862932dc8afa9144c4a5bc5dfe29c2f7dde1b7734175bad49ee310e7'
```

Comment:
Can't this be configured somewhere in GitHub Actions's UI? It's not a sensitive token, but I think it's best to keep tokens outside the source.

Reply:
There is support for secrets somewhere. Is it ok to set it up like this for now and I'll leave an issue?
I find the doc of the github actions really difficult to search through.

Reply:
Yea, that's fine.